multilabel_classification/pytorch_network_training.ipynb

```diff
@@ -0,0 +1,1064 @@
+{
```

Comment:
First cell should always be markdown with a title and short description of what the notebook does.

In this case, title can be:  Training a neural network for multi-label classification on the Celeb-A dataset

Reply:
Done: https://github.com/cleanlab/examples/pull/27/commits/28f62b8528fdc7aec2fd87bca2d5da8e142b7b1e